### ISSUE TITLE
BUG - DevOps - ChromeDriver version compatibility

### ISSUE THREAD

**Issue by IsraelleHub** [department-of-veterans-affairs/va-mobile-app#9965]
## Problem
The accessibility check workflow failed due to chromeDriver version compatibility. We should update the compatible chromeDriver version on the accessibility workflow for it to stop [failing](https://github.com/department-of-veterans-affairs/va-mobile-app/actions/runs/11456853817/job/31875885994)
## Screenshots
<img width="1514" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNlJrQnc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--06b0d9f3ecbaafd075c2f4d5794ccf87511f1486/Screenshot%202024-10-22%20at%2010.46.36%E2%80%AFAM.png" alt="Screenshot 2024-10-22 at 10.46.36 AM.png" />
